### PR TITLE
NFT Actions

### LINKED ISSUE #889: NFT Actions
As a user, I would like a no code way to interact with NFTs.

Obviously, it would be great to be able to list them on Stargaze, but will make a separate issue for that. This is about the basics.

- [x] Transfer
- [x] Send (with smart contract message)
- [x] Mint
- [x] Burn

### PR DESCRIPTION
**Pull request by NoahSaso**
Resolves #889 

## Mint
Screenshot 2022-12-12 at 7 27 32 PM
Screenshot 2022-12-12 at 7 27 49 PM
Screenshot 2022-12-12 at 7 28 15 PM
image

## Transfer/Send
Screenshot 2022-12-12 at 7 29 21 PM
Screenshot 2022-12-12 at 7 29 50 PM
image

## Burn
Screenshot 2022-12-12 at 7 30 14 PM

### CODE REVIEW

**Review by elsehow — APPROVED** [review]
Looks great! A few wee comments.

**Review by JakeHartnell — APPROVED** [review]
Amazing! :shipit:

File: apps/dapp/pages/api/uploadImage.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    })
+
+    form.on('end', () => resolve(files))
+    form.on('error', (err) => reject(err))
+
+    // Parse form.
+    form.parse(req)
+  })
+
+  // Make sure there is only one file.
```
[the remark this answers is not in the document]
Comment (NoahSaso) [reply apps/dapp/pages/api/uploadImage.ts:32]: Good point, will clarify.

File: packages/i18n/locales/en/translation.json [hunk 2]
```diff
@@ -124,6 +127,7 @@
     "clock": "Clock",
     "deposit": "Deposit",
     "family": "Family",
+    "fire": "Fire",
     "gear": "Gear",
     "handshake": "Handshake",
     "herb": "Herb",
```
[the remark this answers is not in the document]
Comment (NoahSaso) [reply packages/i18n/locales/en/translation.json:133]: I'd rather manually enter emoji names like this than include some set of all names. Negligible either way, but including a pre-existing list feels to me like waste.

File: packages/stateful/actions/actions/nft/makeMintNftAction.tsx [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+        Icon={ImageEmoji}
+        onRemove={props.onRemove}
+        title={t('title.mintNft')}
+      >
+        <SuspenseLoader fallback={<Loader />} forceFallback={!mounted}>
+          {contractChosen ? (
+            // If token URI is set, we don't need to upload metadata. If
+            // viewing a created proposal, this is decoded from the cosmos
+            // message. If creating a new proposal, this is set by the
+            // `UploadNftMetadata` component once the metadata is uploaded.
```
[the remark this answers is not in the document]
Comment (NoahSaso) [reply packages/stateful/actions/actions/nft/makeMintNftAction.tsx]: Will improve comment 🫡

**Review by elsehow — APPROVED** [review]

**Pull request merged**